tools/docker/static-analysis/cppcheck.sh

```diff
@@ -31,6 +31,7 @@ run_cppcheck() {
              --error-exitcode=1 \
              --enable=warning,style,information \
              -j"$(nproc)" \
+             --inline-suppr \
```

Comment:
If we follow @tomereli's advice on working around the problem rather than suppressing it, we also should remove this commit.